Dockerfile

```diff
@@ -4,13 +4,15 @@ USER root
 # Volume for sharing wrapper script
 VOLUME /utils
 RUN apt-get update
-RUN apt-get -y install apt-utils python wget build-essential python-dev swig libevent-dev python-protobuf libprotobuf-dev libcurl4-gnutls-dev python-setuptools libxml2-dev libxslt-dev libblas-dev liblapack-dev gfortran libfreetype6-dev libpng-dev
-
-RUN wget http://arshaw.com/scrapemark/downloads/scrapemark-0.9-py2.7.egg; easy_install scrapemark-0.9-py2.7.egg
+RUN apt-get -y install apt-utils python wget build-essential python-dev swig libevent-dev python-protobuf libprotobuf-dev libcurl4-gnutls-dev libxml2-dev libxslt-dev libblas-dev liblapack-dev gfortran libfreetype6-dev libpng-dev
 
 # Install pip
 RUN wget https://bootstrap.pypa.io/get-pip.py; python get-pip.py
 
+RUN wget http://arshaw.com/scrapemark/downloads/scrapemark-0.9-py2.7.egg; easy_install scrapemark-0.9-py2.7.egg
+
+RUN pip install distribute
```

Comment:
Ok so tl;dr - I think we should bump `python-stdnum` to 1.0 to pick up arthurdejong/python-stdnum/commit@c3d669ca074557bcf57a4742b83d178364b08465 to make `distribute` not be required.

Long version: I watched the build output and noticed this line:

> Collecting distribute (from python-stdnum==0.7->-r /etc/requirements.txt (line 40))

So that made me realise `distribute` was getting pulled in because of `python-stdnum`. So I looked in `requirements.txt` to see what was going on, and noticed it says this:

> \# The following packages are considered to be unsafe in a requirements file:
\# distribute                # via python-stdnum
[...]

So that made me guess that given it's "unsafe" there's probably a later version of `python-stdnum` which removes the dependency, which led me to the above commit.

Changed to `1.0` in `requirements.in`, ran `pip-compile --output-file requirements.txt requirements.in` - the `distribute` line did indeed disappear. Rebuilt docker image - it worked.

The only remaining question is whether it's ok to bump to `1.0`. Given what the library does ("parse, validate and reformat standard numbers and codes in different formats") and the fact this is very, very peripheral to OpenCorporates, I think it's fine.

I will update the branch 👍